scorescreen.py

```diff
@@ -21,98 +21,112 @@
-                # totaltime+=timer.tick()
                 if event.type == pygame.QUIT:
-                    crashed = True
-                if event.type == pygame.KEYDOWN:
-                    # swoosh.play(0)
-                    return 1
+                    self.running = False
+                if event.type == pygame.MOUSEBUTTONUP and event.button == 1:
+                    left_click_pressed = True
 
             mos_x, mos_y = pygame.mouse.get_pos()
-            g.gameDisplay.fill(white)
-            g.gameDisplay.blit(scorescreen, (350, 0))
-
-            msg = font3.render(_("GAME OVER"), 2, white)
-            g.gameDisplay.blit(msg, (350, 120))
-            scoress = font2.render(_("SCORE      ") + _(str(scores)), 2, white)
-            g.gameDisplay.blit(scoress, (500, 265))
-            scoress = font2.render(_("BEST        ") +
-                                   _(str(maxscore)), 2, white)
-            g.gameDisplay.blit(scoress, (510, 330))
-
-            if restart.get_rect(center=(550 + 60, 420 + 60)).collidepoint(mos_x, mos_y):
-                g.gameDisplay.blit(pygame.transform.scale(
-                    restart, (124, 124)), (550 - 2, 420 - 2))
-                if(pygame.mouse.get_pressed())[0] == 1:
+            game.screen.fill(BLACK)
+            game.screen.blit(self.scorescreen, (350, 0))
+
+            msg = self.font3.render(_("GAME OVER"), 2, WHITE)
+            game.screen.blit(msg, (350, 120))
+            scoress = self.font2.render(_("SCORE      ") +
+                                        _(str(scores)), 2, WHITE)
+            game.screen.blit(scoress, (500, 265))
+            scoress = self.font2.render(_("BEST        ") +
+                                        _(str(max_score)), 2, WHITE)
+            game.screen.blit(scoress, (510, 330))
+
+            if self.restart.get_rect(
+                    center=(550 + 60, 420 + 60)).collidepoint(mos_x, mos_y):
+                game.screen.blit(pygame.transform.scale(
+                    self.restart, (124, 124)), (550 - 2, 420 - 2))
+                if left_click_pressed:
                     return 1
             else:
-                g.gameDisplay.blit(restart, (550, 420))
+                game.screen.blit(self.restart, (550, 420))
 
-            # left and right black background patches
-            pygame.draw.rect(g.gameDisplay, black, (0, 0, 350, 768))
-            pygame.draw.rect(g.gameDisplay, black, (840, 0, 693, 768))
             pygame.display.update()
-            clock.tick(60)
-
-            if crashed == True:                       # Game crash or Close check
-                pygame.quit()
-                sys.exit()
-
-        # Just a window exception check condition
-        event1 = pygame.event.get()
-        if event1.type == pygame.QUIT:
-            crashed = True
-
-        if crashed == True:
-            pygame.quit()
-            sys.exit()
+            clock.tick(FPS)
+        pygame.quit()
+        sys.exit()
```

Comment:
Please get rid of these too and use a return.

Reply:
Fixed